ScoutSuite/frontend/src/components/Partial/PartialValue/index.js

```diff
@@ -61,10 +64,12 @@ const PartialValue = props => {
   const setIssueLevel = useContext(PartialTabContext);
 
   const fullValuePath = concatPaths(basePath, valuePath);
-  const value = renderValue(props.value || get(ctx.item, fullValuePath));
+  let value = renderValue(
+    props.value || get(ctx.item, fullValuePath, props.value)
+  );
 
-  if (value === undefined || value === null) {
-    return null;
+  if (typeof value === 'boolean') {
+    value = String(value);
```

Comment:
I added this so we wouldn't need to add a toString() method in renderValue everytime the value is a bool.